spec/services/quarterly_message_integration_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "business_time"
+
+# I really hate putting this test here as it does not correspond
```

Comment:
I agree that this is faster than a feature spec but I propose we do one of the following:

1. create a class that encapsulates this behavior so we can test that class in order to ensure that everything is working together, like you say
OR
2. just make it a feature spec. would be slower (like you say), but having this type of random integration spec floating around makes it harder for future developers to know what types of tests they need.